app/Http/routes.php

```diff
@@ -75,8 +75,24 @@
 /*-----------------------*
  *  Profile Controller   *
  *-----------------------*/
- 
- 
+
+Route::get('profile/view/{username}'		, 'ProfileController@profileView')		->where('username', '([a-zA-Z]{1,})([0-9]+)?\w+');	// Implemented
```

Comment:
This validation shouldn't be done here. Usernames with illegal characters won't be created, and will 404 and that's totally fine.